controller.go

```diff
@@ -1363,3 +1364,27 @@ func (c *controller) IsDiagnosticEnabled() bool {
 	defer c.Unlock()
 	return c.DiagnosticServer.IsDiagnosticEnabled()
 }
+
+func (c *controller) iptableEnabled() bool {
+	c.Lock()
+	defer c.Unlock()
+
+	if c.cfg == nil {
+		return false
+	}
+	// parse map cfg["bridge"]["generic"]["EnableIPTable"]
+	cfgBridge, ok := c.cfg.Daemon.DriverCfg["bridge"].(map[string]interface{})
```

Comment:
1. just want to be safe, in case iptableEnable() is called beore makeDriverConfig()
2. ideally it could have been global option. alas, "EnableIPTable" bool is stuff in bridge configure.